tests/abstract/class.tx_mklib_abstract_ObservableT3ServiceTest.php

```diff
@@ -23,14 +23,12 @@
  *  This copyright notice MUST APPEAR in all copies of the script!
  */
 
-tx_rnbase::load('tx_mklib_tests_fixtures_classes_ObservableT3Service');
```

Comment:
Please require this file with require_once \tx_rnbase_util_Extensions::extPath('mklib').'tests/fixtures/classes/class.tx_mklib_tests_fixtures_classes_ObservableT3Service.php';. There seems to be a problem with autoloading but this shouldn't be a problem for tests. After that please check again if the test case is fine or the markTestIncomplete is still needed.

Reply:
It was a problem with the autoloading.

I added the ` tx_rnbase::load('tx_mklib_tests_fixtures_classes_ObservableT3Service'); `
 again. 

I also added this to the composer.json
```
    "autoload-dev": {
        "classmap": [
            "tests/"
        ]
    },
```

That solved the problem.

Reply:
Yeah, you're right. Tx_Mklib_Domain_Model_Iso_Iban are Tx_Mklib_Domain_Model_Iso_SwiftBic no constants but class name that are not put into strings.